### PR TITLE
[feature2] Second feature for this project

### PR DESCRIPTION
**Pull request by Pratap-Samal**
Just added new feature2
	1. New folder created
	2. New file was added into the folder

### CODE REVIEW

File: another-feature.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Some content on another-file.txt
+
+adding second line content
```
Comment (Kumar-Samal) [inline another-feature.txt:3]: This line was required.

**Review by Kumar-Samal — APPROVED** [review]
It looks good, you can proceed

File: another-feature.txt (hunk 1 above)
Comment (Pratap-Samal) [reply another-feature.txt:3]: Thanks..

**Comment by Pratap-Samal** [comment]
Finalizing feature2, merging into main branch

**Pull request merged**